concepts/05 UI Components/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,180 @@
+You can change any form, editor, or item properties at runtime. In **jQuery**, use the [option(optionName, optionValue)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#optionoptionName_optionValue) method to update a Form property, and the [itemOption(id, option, value)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#itemOptionid_option_value) to update the value of an item property. In **Angular**, **Vue** or **React**, bind the property's value to a component property.
```

Comment:
When you need to give different descriptions for different frameworks, put the descriptions under accordion tabs, as we do in the Create a Form piece.